src/graphql/root/query/globals.ts

```diff
@@ -2,12 +2,17 @@ import { GT } from "@graphql/index"
 import Globals from "@graphql/types/object/globals"
 import { getActiveOnchainLnd } from "@services/lnd/utils"
 
+const commitHash = process.env.COMMITHASH
```

Comment:
all our app configuration info are in `app.ts`, I'll suggest to move those values there

Reply:
Fixed https://github.com/GaloyMoney/galoy/commit/07a21c879108e1ec812b5a8cd87f2fc3dfb0bbe1